### PR TITLE
Api facility

### PR DESCRIPTION
**Pull request by abinmn**
- Add DRF to installed apps
- Create endpoints related to the facility model.
- `is_active` field is not included in API response.

### CODE REVIEW

**Comment by vigneshhari** [comment]
verified is also an internal tool , dont expose it

**Comment by vigneshhari** [comment]
The user should only be able to see the facilities he created, also paginate everything

**Comment by tomahawk-pilot** [comment]
@abinmn fix your merge conflicts

File: config/urls.py [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 from config import api_router
 
+from config import api_router
```
Comment (afzalSH) [inline config/urls.py]: Duplicate import

**Comment by biswaz** [comment]
Closing this. Your commits will still be present when I merge #55 @abinmn

**Comment by abinmn** [comment]
Okay.

**Pull request closed without merging**